lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,10 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
 <div class="recent-updates">
   % for index, update in enumerate(visible_updates):
     <article class="updates-article">
       % if not update.get("is_error"):
-        <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
+        <h5 class="date" id="msg-date-${index}">${update.get("date")}</h5>
```

Comment:
I think article works here. From HTML5 spec:

> The article element represents a complete, or self-contained, composition in a document, page, application, or site and that is, in principle, independently distributable or reusable, e.g. in syndication. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.

I also don't have any issue with h5's as long as the order is logical (RE as long as the parent is an h4)